OpenTESArena/src/Game/Game.cpp

```diff
@@ -36,7 +34,8 @@ Game::Game()
 	this->options = OptionsParser::parse();
 
 	// Initialize virtual file system using the Arena path in the options file.
-	VFS::Manager::get().initialize(std::string(this->options->getArenaPath()));
+    std::string arena = File::toString(this->options->getArenaPath());
```

Comment:
Ah, didn't realize that. I guess I was thinking of [Java's File.toString()](http://docs.oracle.com/javase/8/docs/api/java/io/File.html#toString--), which returns the file path. Really what I'm wanting to do is take advantage of the check to see if the file actually exists.